cmd/kubelet/app/options/options.go

```diff
@@ -114,6 +112,7 @@ func NewKubeletServer() *KubeletServer {
 			RktPath:                        "",
 			RktStage1Image:                 "",
 			RootDirectory:                  defaultRootDir,
+			RuntimeContainer:               "/docker-daemon",
```

Comment:
The assumption is that this fixes systemd if and only if you set this flag to "/system.slice/docker.service"?

  It looks like the default value is always wrong for systemd.  Maybe I need to look more as I review PR